models/payment.go

```diff
@@ -25,7 +25,7 @@ type PaymentResourceData struct {
 	CreatedBy               CreatedBy      `json:"created_by"                          bson:"created_by"`
 	Description             string         `json:"description"                         bson:"description"`
 	Links                   Links          `json:"links"                               bson:"links"`
-	PaymentMethod           string         `json:"payment_method,omitempty"            bson:"payment_method,omitempty"`
+	PaymentMethod           string         `json:"payment_method"                      bson:"payment_method"`
```

Comment:
According to the spec, `payment_method` is an optional field, so why has `omitempty` been removed?

Reply:
The patch request won't work updating the field unless there is already a value present

Reply:
Following discussion the json can be left as `omitempty` whilst it's removed for the bson and this should still work whilst adhering to the spec